src/background/requestHandler.js

```diff
@@ -42,11 +42,11 @@ export class RequestHandler extends Component {
     });
 
     propertySum(
```

Comment:
Very snazzy. I think we can tidy RequestHandler up nicely (in a future PR) by adding `this.extState` to this propertySum and consolidating `handleExtensionStateChanges` and `updateProxyInfoFromClient`.